x-pack/plugins/security/public/management/management_urls.ts

```diff
@@ -4,19 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const MANAGEMENT_PATH = '/management';
-export const SECURITY_PATH = `${MANAGEMENT_PATH}/security`;
+const MANAGEMENT_PATH = '/management';
```

Comment:
__note:__ don't have good ideas on how to reorganize this yet since we must obey the `basepath` provided during `mount` phase now. So we kind of have two sources of management URLs now. I decided to keep it as is in this PR and think about it later.

Reply:
Rather than having static exports, we could construct a management url service/class/object which accepts the `basePath` provided during the `mount` phase, and build the urls off of that.

I'm also not opposed to hard-coding the expected URLs in the functional tests to eliminate the dependency here.

The `get${entity}Href` functions below are used within our NP management apps, so they aren't honoring the provided `basePath` right now either.

Reply:
> Rather than having static exports, we could construct a management url service/class/object which accepts the basePath provided during the mount phase, and build the urls off of that.

The problem with this approach is that it only works in the scope of a single app because basepath is only available at `mount` for this app (basepath already includes app id). So to navigate from Role Mappings app to Roles you'll need to rely on hardcoded URLs unless I didn't get the idea? 

That's why I was thinking to wait and see how `navigateToApp` provided by Management will work (currently it's just core's original `navigateToApp` that does "global" navigation, not within Management app) and whether we'll have something like management specific `getAppBasePath` as well.